src/cargo/core/manifest.rs

```diff
@@ -222,7 +222,7 @@ impl TargetKind {
     }
 }
 
-/// Information about a binary, a library, an example, etc. that is part of the
+/// Information about a binary, a library, an example, etc. That is part of the
```

Comment:
I don't think this is really a typo.  Deconstructing:

> Information about a _thing_ that is part of the package.

The second part isn't a separate sentence.

Reply:
Indeed, this is not typo as you pointed out. I remove this commit from PR.